src/app/issue/summary/summary.component.ts

```diff
@@ -0,0 +1,152 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+import { AuthService } from 'src/app/auth/auth.service';
+import { BaseService } from 'src/app/base.service';
+import { Issue } from 'src/app/interfaces/issue/Issue';
+import { IssueRequirement } from 'src/app/interfaces/issue/IssueRequirement';
+import { User } from 'src/app/interfaces/User';
+import { IssueRequirementsService } from '../issue-requirements.service';
+import { IssueSummaryService } from '../issue-summary.service';
+import { IssueService } from '../issue.service';
+
+@Component({
+  selector: 'app-summary',
+  templateUrl: './summary.component.html',
+  styleUrls: ['./summary.component.less']
+})
+export class SummaryComponent implements OnInit {
+
+  httpOptions = {
+    headers: new HttpHeaders({
+      'Content-Type': 'application/json',
+    }),
+  };
+
+  constructor( private issueRequirementsService: IssueRequirementsService,
+    private route: ActivatedRoute,
+    private base: BaseService,
+    private httpClient: HttpClient,
+    private authService: AuthService,
+    private issueService: IssueService,
+    private router: Router,
+    private issueSummaryService: IssueSummaryService) { }
+
+  ngOnInit(): void {
+    this.projectId = this.route.snapshot.paramMap.get('projectId'); 
+    this.companyId = this.route.snapshot.paramMap.get('companyId'); 
+    this.getAllRequirements();
+    this.currentUser = this.authService.currentUserValue;
+  }
+
+
+  currentUser: User;
+
+  listOfRequirements: IssueRequirement[];
+
+  issueId: string;
+  projectId: string;
+  companyId: string;
+  currentIssue: Issue;
+
+  expectedTime: number;
+
+  getAllRequirements(){
+    this.issueId = this.route.snapshot.paramMap.get('issueId'); 
+    this.issueRequirementsService.getRequirements(this.issueId).subscribe(
+      (data) => {
+        this.listOfRequirements = data;
+
+      },
+      (error) => {
+        // TODO Fehlerausgabe
+        console.error(error);
+      }
+    );
+    this.issueService.getIssue(this.projectId, this.issueId).subscribe(
+      (data) => {
+        this.currentIssue = data;
+
+      },
+      (error) => {
+        // TODO Fehlerausgabe
+        console.error(error);
+      }
+    );
+  }
+
+  removeRequirement(req: IssueRequirement){
+    this.issueRequirementsService.deleteRequirement(this.route.snapshot.paramMap.get('issueId'), req.id).subscribe(
+      (data)=>{
+        this.router.navigateByUrl(`${this.companyId}/projects/${this.projectId}/issues/${this.issueId}`);
+      },
+      (error) =>{
+        console.error(error);
+      }
+    );
+  }
+
+
+
+  sendSummary(
+  ): Observable<any> {
+    let issue: any;
```

Comment:
Musst du das Issue updaten? ``requirementsAccepted`` wird glaube ich auch vom Backend gesetzt.

Reply:
Soweit ich es im Backend erkennen konnte, wird in der CreateSummary-Methode nur überprüft, ob im IssueDetail die Anzahl der Requirements und die expectedTime größer als Null sind, und daraufhin wird RequirementsSummaryCreated auf true gesetzt und ein ConversationItem hinzugefügt. Insofern muss vor dem erstellen der Summary an irgendeinem Punkt das Issue mit der eingegebenen expectedTime geupdated werden,  was momentan direkt davor passiert. Hab ich da etwas übersehen oder falsch verstanden?